packages/contracts-bedrock/test/L2/OptimismSuperchainERC20.t.sol

```diff
@@ -0,0 +1,323 @@
+        vm.expectEmit(true, true, true, true, address(superchainERC20));
+        emit IERC20.Transfer(_from, ZERO_ADDRESS, _amount);
+
+        // Look for the emit of the `Burn` event
+        vm.expectEmit(true, true, true, true, address(superchainERC20));
+        emit OptimismSuperchainERC20.Burn(_from, _amount);
+
+        // Call the `burn` function with the bridge caller
+        vm.prank(BRIDGE);
+        superchainERC20.burn(_from, _amount);
+
+        // Check the total supply and balance of `_from` after the burn were updated correctly
+        assertEq(superchainERC20.totalSupply(), _totalSupplyBefore - _amount);
+        assertEq(superchainERC20.balanceOf(_from), _fromBalanceBefore - _amount);
+    }
+
+    /// @notice Tests the `sendERC20` function burns the sender tokens, sends the message, and emits the `SentERC20`
+    /// event.
+    function testFuzz_sendERC20_succeeds(address _sender, address _to, uint256 _amount, uint256 _chainId) external {
+        // Ensure `_sender` is not the zero address
+        vm.assume(_sender != ZERO_ADDRESS);
+        vm.assume(_to != ZERO_ADDRESS);
+
+        // Mint some tokens to the sender so then they can be sent
+        vm.prank(BRIDGE);
+        superchainERC20.mint(_sender, _amount);
+
+        // Get the total supply and balance of `_sender` before the send to compare later on the assertions
+        uint256 _totalSupplyBefore = superchainERC20.totalSupply();
+        uint256 _senderBalanceBefore = superchainERC20.balanceOf(_sender);
+
+        // Look for the emit of the `Transfer` event
+        vm.expectEmit(true, true, true, true, address(superchainERC20));
+        emit IERC20.Transfer(_sender, ZERO_ADDRESS, _amount);
+
+        // Look for the emit of the `SentERC20` event
+        vm.expectEmit(true, true, true, true, address(superchainERC20));
+        emit OptimismSuperchainERC20.SentERC20(_sender, _to, _amount, _chainId);
+
+        // Mock the call over the `sendMessage` function and expect it to be called properly
+        bytes memory _message = abi.encodeCall(superchainERC20.relayERC20, (_to, _amount));
+        _mockAndExpect(
+            MESSENGER,
+            abi.encodeWithSelector(
+                IL2ToL2CrossDomainMessenger.sendMessage.selector, _chainId, address(superchainERC20), _message
+            ),
+            abi.encode("")
+        );
+
+        // Call the `sendERC20` function
+        vm.prank(_sender);
+        superchainERC20.sendERC20(_to, _amount, _chainId);
+
+        // Check the total supply and balance of `_sender` after the send were updated correctly
+        assertEq(superchainERC20.totalSupply(), _totalSupplyBefore - _amount);
+        assertEq(superchainERC20.balanceOf(_sender), _senderBalanceBefore - _amount);
+    }
+
+    /// @notice Tests the `sendERC20` function reverts when the `_to` address is the zero address.
+    function testFuzz_sendERC20_zeroAddressTo_reverts(uint256 _amount, uint256 _chainId) public {
```

Comment:
minor nitpick, in previous cases you are ordering reverts first but for `sendERC20` they it is after the succeed case, can you reorder it so they all follow the same pattern?